networkit/cpp/centrality/Betweenness.cpp

```diff
@@ -30,20 +30,9 @@ void Betweenness::run() {
         edgeScoreData.resize(z2);
     }
 
-    // thread-local scores for efficient parallelism
-    count maxThreads = omp_get_max_threads();
-    std::vector<std::vector<double> > scorePerThread(maxThreads, std::vector<double>(G.upperNodeIdBound()));
-    DEBUG("score per thread: ", scorePerThread.size());
-    DEBUG("G.upperEdgeIdBound(): ", G.upperEdgeIdBound());
-    std::vector<std::vector<double> > edgeScorePerThread;
-    if (computeEdgeCentrality) {
-        edgeScorePerThread.resize(maxThreads, std::vector<double>(G.upperEdgeIdBound()));
-    }
-    DEBUG("edge score per thread: ", edgeScorePerThread.size());
-
-    std::vector<std::vector<double>> dependencies(maxThreads, std::vector<double>(z));
+    std::vector<std::vector<double>> dependencies(omp_get_max_threads(), std::vector<double>(z));
     std::vector<std::unique_ptr<SSSP>> sssps;
-    sssps.resize(maxThreads);
+    sssps.resize(omp_get_max_threads());
```

Comment:
Similar to #607, the `SSSP` instance could instead be initialized here without the global `sssps` vector.